src/main/java/baron/commands/UnmarkTaskCommand.java

```diff
@@ -5,22 +5,41 @@
 import baron.tasks.TaskManager;
 import baron.util.Storage;
 
+/**
+ * Represents a unmark task command that unmarks the task during execution.
+ */
 public class UnmarkTaskCommand extends Command {
     private final TaskManager taskManager;
     private final Storage storage;
-    private final String commandArgs;
+    private final String commandArg;
```

Comment:
I like how you change the name of ``commandArgs`` to ``commandArg`` since ``commandArg`` only represents a single command

Reply:
Thank you! :)